openerp/addons/base/ir/ir_qweb.py

```diff
@@ -1220,53 +1214,70 @@ def checksum(self):
         check = self.html + str(self.last_modified)
         return hashlib.sha1(check).hexdigest()
 
-    def js(self):
-        content = self.get_cache('js')
-        if content is None:
-            content = ';\n'.join(asset.minify() for asset in self.javascripts)
-            self.set_cache('js', content)
-        return content
+    def clean_attachments(self, type):
+        ira = self.registry['ir.attachment']
+        domain = [('url', 'not like', '/web/content/%%-%s/%%' % self.version), ('url', 'like', '/web/content/%%-%%/%s%%.%s' % (self.xmlid, type))]
```

Comment:
should use `=like` operator (prepend a `!` operator for negate it)

same for `get_attachments`